FirebaseAdmin/FirebaseAdmin/Messaging/WebpushConfig.cs

```diff
@@ -44,6 +44,12 @@ public sealed class WebpushConfig
         [JsonProperty("notification")]
         public WebpushNotification Notification { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Webpush options that will be included in the message.
+        /// </summary>
+        [JsonProperty("fcm_options")]
+        public WebpushFcmOptions Options { get; set; }
```

Comment:
I missed this in my previous review. Lets rename this property to `FcmOptions` to be consistent with other languages (namely Node.js and Go).